crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -367,26 +363,6 @@ async fn test_publish_gas() -> anyhow::Result<()> {
         expected_gas_balance,
     );
 
-    // Create a transaction with gas_budget that's 1 less than the amount needed to
```

Comment:
this was not doing what the comment was saying and removing the min charge makes those tests hard.
We should have them but not how this is